src/ContainsDuplicates.java

```diff
@@ -0,0 +1,18 @@
+public class ContainsDuplicates {
+//    public static void main(String[] args) {
+//        int[] arr = {1, 2, 3, 4, 6, 6, 7, 8, 9, 10};
+//        containsDuplicates(arr);
+//    }
+
+    public static boolean containsDuplicates(int[] arr) {
```

Comment:
This is totally valid and you do not need to change anything - but there is a way to do it without the nested `for loops`  :)